src/test/java/org/minions/devfund/angela/moviedatabase/MovieDatabaseTest.java

```diff
@@ -0,0 +1,102 @@
+package org.minions.devfund.angela.moviedatabase;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Test for {@link MovieDatabase}.
+ */
+public class MovieDatabaseTest {
+
+    private MovieDatabase movieDatabase;
+    private static final String MOVIE_NAME = "Infinity War";
+    private static final String BEST_ACTOR = "Robert Downey";
+    /**
+     * Preconditions.
+     */
+    @Before
+    public void setup() {
+        movieDatabase = new MovieDatabase();
+        movieDatabase.addMovie(MOVIE_NAME, new String[]{BEST_ACTOR, "Chris Hemsworth", "Chris Evans"});
+    }
+
+    /**
+     * Verifies all actors where added.
+     */
+    @Test
+    public void testActorsAdded() {
+        final List<Actor> actors = new ArrayList<>();
+        actors.add(new Actor(BEST_ACTOR));
+        actors.add(new Actor("Chris Hemsworth"));
+        actors.add(new Actor("Chris Evans"));
```

Comment:
Is it really need to have list of Actors ?
o just to have an array of strings ?

Reply:
Done